Classes/Domain/Model/ApplicationB.php

```diff
@@ -257,7 +257,7 @@ public function getCountry()
      * @param SJBR\StaticInfoTables\Domain\Model\Country $country
      * @return void
      */
-    public function setCountry(Country $country)
+    public function setCountry(Country $country = null)
```

Comment:
Don't we need the same in `setNationality()`?

Reply:
Nationality is a string - which seems to convert fine from an empty form field.

You ask why it is a string? Legacy.... ;)

Reply:
Ah yeah, you mentioned that recently.